### ISSUE TITLE
Shiny app for data tables -- unclear

### ISSUE THREAD

**Issue by ToonTalk** [ox-it/LiveData_htmlwidgets#53]
I didn't understand 

> Select rows in the interactive table and see the row number displayed in the app

The row numbers are displayed and how is Shiny helping here since DT can display row numbers without any help from Shiny.

Perhaps a better motivated Shiny app where the server is doing something more important would be better.

**Comment by charliejhadley** [comment]
Agreed - need a better example.. something like a table of pokemon and selecting a row in the shiny app will show a picture of the pokemon elsewhere in the app.

It's important to not create to trivial an example of a datatable, but I think that most of the time datatables will be used for one of two purposes:
1. Selecting a data point shows a datatable
2. User wants to just show stuff in a datatable and optionally add/remove columns

Should also repeat that if you don't need a shiny app can just use RMarkdown workflow to generate requisite HTML/JavaScript and host somewhere static (like Github pages!)
